src/trodes_to_nwb/convert_position.py

```diff
@@ -785,13 +785,21 @@ def add_position(
 
     epoch_to_camera_ids = pd.concat(df).set_index("epoch").sort_index()
 
-    position = Position(name="position")
-
     # Make a processing module for behavior and add to the nwbfile
     if not "behavior" in nwb_file.processing:
         nwb_file.create_processing_module(
             name="behavior", description="Contains all behavior-related data"
         )
+
+    new_position = (
+        "position" not in nwb_file.processing["behavior"].data_interfaces.keys()
```

Comment:
If data_interfaces is a dict, then you don't need to call .keys().
Also could just put this directly in the conditional.